news/groups_in_people_schema.rst

```diff
@@ -0,0 +1,13 @@
+**Added:** None
+
+**Changed:**
+
+ENH: added group item in people schema
```

Comment:
This needs `* ` before the text, also you don't need the `ENH:` for this (although it is ok if you want).

Reply:
ok, fixed in the next commit